packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/to/helpers/V1_DatabaseBuilderHelper.ts

```diff
@@ -161,6 +161,15 @@ const schemaExists = (database: Database, _schema: string): boolean =>
   DEFAULT_DATABASE_SCHEMA_NAME === _schema ||
   _schemaExists(database, _schema, new Set<Database>());
 
+export const V1_assertSchemaExists = (
```

Comment:
To be consistent, let's rename this like the rest

```suggestion
export const V1_findSchema = (
```